### ISSUE TITLE
Could not login some web services

### ISSUE THREAD

**Issue by leico** [takedajs/TitleUrlCopy#15]
macOS Mojave 10.14.6
Firefox 77.0.1

Hello developer. Thank you for publish it. This pugin is so useful. I heavily use this when I write markdown notes.
But, I encountered  a problem. When enable this plugin, I could not login some web services.
I encountered above ploblem in those services.
 
* <https://my.netgear.com>
* <https://note.com>

Thanks to read.

**Comment by uKLEina** [comment]
The same issue happened to me.

OS: Windows 10 Version 1909
Firefox: Version 80.0.1

I tried to log in to note.com, but I couldn't. I turned off TitleUrlCopy and then I could. So I think something is wrong with this extension.
I don't know if it helps but the console error message is below:

```
Uncaught Error: Permission denied to access property "apply"
    trigger https://d2l930y2yx77uc.cloudfront.net/assets/library-71e1e770971f84e2ac238c03ce794a321a66912c1fc27d36b4fffbcda16b3919.js:2
    trigger https://d2l930y2yx77uc.cloudfront.net/assets/library-71e1e770971f84e2ac238c03ce794a321a66912c1fc27d36b4fffbcda16b3919.js:2
    each https://d2l930y2yx77uc.cloudfront.net/assets/library-71e1e770971f84e2ac238c03ce794a321a66912c1fc27d36b4fffbcda16b3919.js:1
    each https://d2l930y2yx77uc.cloudfront.net/assets/library-71e1e770971f84e2ac238c03ce794a321a66912c1fc27d36b4fffbcda16b3919.js:1
    trigger https://d2l930y2yx77uc.cloudfront.net/assets/library-71e1e770971f84e2ac238c03ce794a321a66912c1fc27d36b4fffbcda16b3919.js:2
    <anonymous> https://d2l930y2yx77uc.cloudfront.net/assets/application-10d21bc9e314ccde2d00bf7ef814437d9e48054359f0bd791ff666ea09fabf4c.js:8
    dispatch https://d2l930y2yx77uc.cloudfront.net/assets/library-71e1e770971f84e2ac238c03ce794a321a66912c1fc27d36b4fffbcda16b3919.js:2
    handle https://d2l930y2yx77uc.cloudfront.net/assets/library-71e1e770971f84e2ac238c03ce794a321a66912c1fc27d36b4fffbcda16b3919.js:2
```

**Comment by takedajs** [comment]
@leico @uKLEina 

Thank you for your report. I'm sorry for the late reply. 
We will confirm the event and consider how to deal with it.

**Comment by takedajs** [comment]
@uKLEina @leico 
Fixed the error. Please upgrade the add-on to 1.4.1 and log in.
Sorry for the late response.

TitleUrlCopy
https://addons.mozilla.org/ja/firefox/addon/titleurlcopy/

**Comment by uKLEina** [comment]
It's confirmed that the problem is solved with 1.4.1.
Thank you very much!

**Comment by leico** [comment]
@takedajs 
Also I have confirmed that the problem has been fixed!
Thank you!
